tools/toolchains/arm.py

```diff
@@ -81,6 +81,12 @@ def __init__(self, target, notify=None, macros=None,
             if "--library_type=microlib" not in self.flags['common']:
                 self.flags['common'].append("--library_type=microlib")
 
+        if (
```

Comment:
This test seems to be duplicated, can this consolidated into a function?

Reply:
[DOne](https://github.com/ARMmbed/mbed-os/compare/c3442d335ffc81df17ff3849ab2969e83998f5c9..074bab34e361c1eebb64aa4aecd9d92903afabb6)